showthread.php

```diff
@@ -1594,6 +1610,8 @@
 		$thread_deleted = 1;
 	}
 
+	$visible_replies = $thread['replies'] - $thread['deletedposts'];
```

Comment:
Why didn't you reduce unapproved posts?

Reply:
Because unapproved posts are visible to moderators.

Reply:
Aren't soft deleted posts also visible to them?

Reply:
I can't remember what I did 6 month's ago. I'll take a quick review and rectify if required.